traffic-sniffer/lib/request-metadata/sniffer/mock/exceptions.ts

```diff
@@ -0,0 +1,2 @@
+export class MockNotFoundError extends Error { }
+export class MockAlreadyExists extends Error { }
```

Comment:
Newline?

Reply:
I don't think that a new line is necessary here